### ISSUE TITLE
Itokawa.glb is not displayed

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#10]
I tried to display [Itokawa.glb](https://github.com/cx20/jsdo-static-contents/blob/master/models/gltf/2.0/Itokawa/glTF-Draco/Itokawa.glb) with RedCube.js v2.0.7.
However, it was not displayed. Itokawa.glb is a model of asteroids compressed in Draco format.

[RedCube.js + Itokawa.glb result:](http://jsdo.it/cx20/OJQx)
```
GL ERROR :GL_INVALID_OPERATION : glDrawElements: attempt to access out of range vertices in attribute 6
```
[Three.js + Itokawa.glb result:](http://jsdo.it/cx20/gykb)
![image](https://user-images.githubusercontent.com/1833979/50907639-35e88d80-146b-11e9-981d-19ffeee8597b.png)

**Comment by Reon90** [comment]
@cx20 was fixed in 2.0.8

**Comment by cx20** [comment]
I confirmed that asteroids are displayed in the latest version.
[RedCube.js 2.0.8 + Itokawa.glb result:](http://jsdo.it/cx20/wjUa)
![image](https://user-images.githubusercontent.com/1833979/50914367-c1691b00-1479-11e9-95d3-a3c6fd4a57f1.png)
